dts/bindings/serial/zephyr,cdc-acm-uart.yaml

```diff
@@ -21,3 +21,7 @@ properties:
     default: 1024
     description: |
       Size of the virtual CDC ACM UART RX FIFO
+
+  label:
+    description:
+      String to set as iInterface for the specific CDC-ACM instance
```

Comment:
iInterface is just an index, I suggest "The string defined by the label property is also used for the USB interface string descriptor."

Reply:
cool, changed